src/llmcompressor/modifiers/obcq/base.py

```diff
@@ -85,7 +85,8 @@ class SparseGPTModifier(SparsityModifierMixin, Modifier):
     # data pipeline arguments
     sequential_update: Optional[bool] = False  # deprecated
     sequential_targets: Union[str, List[str], None] = None
-    targets: Union[str, List[str], None] = None  # alias sequential_targets
+    targets: Union[str, List[str]] = ["Linear"]
+    ignore: List[str] = Field(default_factory=list)
```

Comment:
We're replicating `ignore` for code clarity?

Reply:
Yes, the goal being that a user can look at a modifier and see its arguments (and therefore how to use it) without having to check all of its inheritances.

This could be less necessary in the future when we have dedicated buildthedocs